doc/release-process.md

```diff
@@ -215,10 +228,12 @@ Codesigner only: Sign the windows binaries:
 Codesigner only: Commit the detached codesign payloads:
 
     cd ~/bitcoin-detached-sigs
-    checkout the appropriate branch for this release series
+    #checkout the appropriate branch for this release series
     rm -rf *
     tar xf signature-osx.tar.gz
     tar xf signature-win.tar.gz
+    #copy the notarization ticket
+    cp dist/Bitcoin-Qt.app/Contents/CodeResoucres osx/dist/Bitcoin-Qt.app/Contents/
```

Comment:
Is "resources" misspelled intentionally?

Reply:
Thanks. Fixed.